src/components/Watches.svelte

```diff
@@ -0,0 +1,79 @@
+<script>
+import Icon from 'fa-svelte'
+import { faTrash } from '@fortawesome/free-solid-svg-icons'
+import { getWatches, deleteWatch } from '../data/watch'
+import { onMount } from 'svelte'
+import FilterTag from './FilterTag.svelte'
+import { labelFor } from '../data/requestFiltering'
+import { fly } from 'svelte/transition'
+
+let watches = []
+let loading = false
+let numVisibleLimited = true
+
+onMount(async () => {
+  try {
+    loading = true
+    watches = await getWatches()  
+  } finally {
+    loading = false
+  }
+})
+
+$: visibleWatches = numVisibleLimited ? firstThree(watches) : watches
+
+const firstThree = array => array.slice(0,3)
+
+const relevantCriteria = watch => Object.keys(watch).filter(key => isCriteriaField(key) && hasValue(watch[key]))
+const isCriteriaField = field => ! ['id', 'name'].includes(field)
+const hasValue = value => value !== null
+
+function valueFor(criteria, watch) {
+  const value = watch[criteria]
+
+  return value.description || value.name || value
+}
+
+async function remove(watch) {
+  await deleteWatch(watch.id)
+
+  watches = watches.filter(w => w !== watch)
```

Comment:
Is there reason to/not-to compare `id`s here, rather than the whole `watch` object?

Reply:
I was thinking identity works better here because we wouldn't necessarily want to remove a watch that didn't come from our list...I don't mind using `id` though so it will sync up closer to the previous backend call...theoretically we could run into an issue if this method received a watch with an id that was that held the same `id` as one in the list of `watches` but was not from that same list...

Reply:
https://github.com/silinternational/wecarry-ui/pull/167/commits/84bb15bed7e70b3e50d56b8e6db04317cd89cc1f